config/field_test.yml

```diff
@@ -25,6 +25,22 @@ experiments:
   #     - user_views_article_four_hours_in_day
   #     - user_views_article_nine_days_in_two_week
   #     - user_views_article_twelve_hours_in_five_days
+  feed_strategy:
+    variants:
+      - original
+      - weighted_query_strategy
+    weights:
+      - 80
+      - 20
```

Comment:
If for whatever reason `weighted_query_strategy` causes a problem, is there a faster way to change the weighting here without deploying code? Is it possible to get this from env vars (with sensible defaults) or some other configuration that allows us to change more quickly than a full code -> commit -> PR -> review -> merge -> CI-> deploy ritual?

Reply:
I'll need to dig into field test to see what options we have.  I'm wondering if this file is open for ERB?

Reply:
Welp…I already accounted for it in the design but didn't think of it in these terms:

> To force the current strategy, you want to use the following: `AB_EXPERIMENT_FEED_STRATEGY=original bin/startup`).

That comes from https://github.com/forem/forem/blob/03fd71b3f60345de5fa0cc2618d869aa40d057bd/app/models/ab_experiment.rb#L48-L67

Reply:
My assumption is that injecting an ENV will require some kind of restart.  But this would help us avoid the code -> commit -> PR -> review -> merge -> CI -> deploy ritual.

(And we could then do that without our hair on fire)

Reply:
Awesome! This makes perfect sense and I definitely did not see that on my first pass 😄

Reply:
I wrote it…and spent 15 minutes looking at field test implementation to see how to circumvent.  But then was like "Wait, jgaskins said ENV"